build_deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE="quay.io/notifications-backend"
```

Comment:
I guess it's missing the group name. Which I guess is still `cloudservices` ?

Reply:
right.. will fix it.